src/gui/main.cpp

```diff
@@ -248,6 +248,29 @@ void setupLogging(const CommandLineOptions &options)
     qCInfo(lcMain) << "Arguments:" << qApp->arguments();
 }
 
+void loadQNetworkInformationBackend()
+{
+#if QT_VERSION >= QT_VERSION_CHECK(6, 3, 0)
+    if (!QNetworkInformation::loadDefaultBackend()) {
+        qCWarning(lcMain) << "Failed to load default backend of QNetworkInformation.";
+        if (!QNetworkInformation::loadBackendByFeatures(QNetworkInformation::Feature::Metered)) {
+            qCWarning(lcMain) << "Failed to load backend of QNetworkInformation by metered feature.";
```

Comment:
If that fails too, I guess we should fall back to at least load a backend that support connectivity changes?